src/util.ts

```diff
@@ -37,5 +37,5 @@ export function castArray<T>(input?: T | T[]): T[] {
 }
 
 export function isProd() {
-  return process.env.NODE_ENV !== 'development'
+  return !['development', 'test'].includes(process.env.NODE_ENV ?? '')
```

Comment:
Can we please try to err on the side of more readable code?

I would much rather see something like `return process.env.NODE_ENV !== 'development && process.env.NODE_ENV !== 'test'`. It's so much easier to parse what exactly that code does and intends to do than something like this.

Reply:
@mdonnalley, @peternhale fyi ^

Reply:
How about?
`return !['development', 'test'].some(mode => mode === (process.env.NODE_ENV ?? '')`

Reply:
I think we can go even simpler. How about?
`return !['development', 'test'].reduce((prev, curr) => curr === (process.env.NODE_ENV ?? '') || prev, false)`